source/_posts/2019-11-20-release-102.markdown

```diff
@@ -112,7 +112,7 @@ Users of Xiaomi Vacuum cleaners can now use this great custom Lovelace card:
 
 - Add Unifi Led ([@florisvdk] - [#27475]) ([unifiled docs]) (new-integration)
 - Almond integration ([@balloob] - [#28282]) ([almond docs]) (new-integration)
-- Speech to Text component ([@pvizeli] - [#28434]) ([demo docs]) ([stt docs]) (new-integration)
+- Speech-to-text component ([@pvizeli] - [#28434]) ([demo docs]) ([stt docs]) (new-integration)
```

Comment:
These are commit messages, we should not change those.

Reply:
Ah, I see! Thank you for the catch!